using-mobile-extensions/adobe-audience-manager/README.md

```diff
@@ -109,21 +142,34 @@ public void onCreate() {
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x) %}
 In your app's `application:didFinishLaunchingWithOptions` function, register the Audience Manager extension with the Mobile Core:
 
+### Swift
+
+```swift
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {  
+  MobileCore.registerExtensions([Audience.self, Identity.self], {
+  MobileCore.configureWith(appId: "yourAppId") 
+ })  
+ ...
+}
+```
+
 ### Objective-C
 
 ```objectivec
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
-   [ACPIdentity registerExtension];
-   [ACPAudience registerExtension];
-   [ACPCore start:nil]
-
-   // Override point for customization after application launch.
-   return YES;
+   [AEPMobileCore registerExtensions:@[AEPMobileAudience.class, AEPMobileIdentity.class] completion:^{
+   [AEPMobileCore configureWithAppId: @"yourAppId"];
+  }];
+  ...
 }
 ```
+{% endtab %}
+
+{% tab title="iOS (ACP 2.x) %}
+In your app's `application:didFinishLaunchingWithOptions` function, register the Audience Manager extension with the Mobile Core:
```

Comment:
Same edit question as above:
Should the function signature be styled as:
`application(_:didFinishLaunchingWithOptions:)`
to follow the Swift conventions for function signatures? [StackOverflow discussion](https://stackoverflow.com/questions/54537403/what-is-the-meaning-of-the-in-swift-function-signatures-and-why-is-there-no-co)